Dockerfile

```diff
@@ -1,11 +1,72 @@
-FROM python:3.11-alpine AS base
-
-# Install base system requirements
-RUN apk add --no-cache mediaconch
+FROM python:3.12-alpine AS base
 
 WORKDIR /code
+ENV BUILD_DIR=/code/build
+
+# Create build directory
+RUN mkdir ${BUILD_DIR}
+
+# Install system requirements and dependencies for MediaConch
+RUN apk add --no-cache \
+    git \
+    automake \
+    autoconf \
+    libtool \
+    make \
+    g++ \
+    zlib \
+    libxml2 \
+    libxslt \
+    libxml2-dev \
+    libxslt-dev \
+    libevent-dev \
+    jansson-dev \
+    sqlite-dev \
+    libzen \
+    libzen-dev \
+    libmediainfo \
+    libmediainfo-dev
+
+# Clone ZenLib, MediaInfoLib and MediaConch_SourceCode
+RUN cd ${BUILD_DIR} && \
+    git clone https://github.com/MediaArea/ZenLib.git && \
+    git clone https://github.com/MediaArea/MediaInfoLib.git && \
+    git clone https://github.com/MediaArea/MediaConch_SourceCode.git
```

Comment:
I think we should clone a specific tag instead of just pulling the base branch: `git clone --branch <tag_name> <repository_url>`

I think the most recent release of each of these would be fine.